lm75bd/lm75bd.c

```diff
@@ -27,7 +27,23 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
+  uint8_t buf[2];
+
+  uint8_t addr = 0x0U;
+  i2cSendTo(devAddr, &addr, 1);
+
+  i2cReceiveFrom(devAddr, buf, 2);
+
+  uint8_t b1 = buf[0];
+  uint8_t b2 = buf[1];
+
+  uint16_t result = ((b1 << 8) | b2) >> 5 ;
   
+    if (result & 0x0400) {
```

Comment:
You shouldn't need this if statement

Reply:
I don't think this was addressed

Reply:
I'm multiplying in two's complement, so I have to check for the negative.